scripts/main.js

```diff
@@ -0,0 +1,57 @@
+let books;
+// eslint-disable-next-line
+if (localStorage.getItem('books') === null) {
+  books = [];
+} else {
+  // eslint-disable-next-line
+  books = JSON.parse(localStorage.getItem('books'))
+}
+
+document.querySelector('form').addEventListener('submit', () => {
+  const bookTitle = document.querySelector('.title').value;
+  const bookAuthor = document.querySelector('.author').value;
+
+  const book = [{ bookTitle, bookAuthor }];
+  books.push(book);
```

Comment:
[OPTIONAL] Consider using methods that don't mutate the array.

<img width="960" alt="Screen Shot 2022-08-02 at 7 52 32" src="https://user-images.githubusercontent.com/82426682/182379242-3dd709a0-513d-48c4-ba96-022342948b31.png">